content/_data/roadmap/roadmap.yml

```diff
@@ -342,6 +342,25 @@ categories:
       It also includes integrations with cloud storage providers.
     link: /sigs/cloud-native
     initiatives:
+    - name: CloudEvents support
+      status: current
+      description: >
+         Microservices are the de facto architecture of the cloud and events are the primary way in which microservices communicate a change of state.
+         CloudEvents is a specification for describing event data in common, standardised formats.
+         The CloudEVents plugin would enable Jenkins support for CloudEvents by allowing events to be emitted in Jenkins pipelines and triggering Jobs on certain events,
+         bringing Jenkins one step closer to interoperability with other technologies in the cloud.
+      link: https://www.jenkins.io/projects/gsoc/2021/project-ideas/cloudevents-plugin/
+      labels:
+      - feature
+    - name: Tekton Client plugin
+      status: current
+      description: >
+         This plugin enables Tekton as the build engine to carry out all the Jenkins Kubernetes builds.
+         Jenkins should be able to run any and all Jobs in Tekton and manage them directly through the Jenkins console,
+         creating the foundation for Jenkins to be a Cloud Native CI/CD platform.
+      link: https://github.com/jenkinsci/tekton-client-plugin
+      labels:
+      - feature
```

Comment:
```suggestion
```

At the governance meeting we agreed to rename the existing Tekton build steps roadmap item